frontend/public/actions/ui.ts

```diff
@@ -10,6 +10,7 @@ import {
   ALL_NAMESPACES_KEY,
   LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY,
 } from '@console/shared/src/constants';
+// import { multiClusterRoutePrefixes } from '@console/app/src/components/detect-cluster/cluster'
```

Comment:
Using the import caused most of the tests to fail.  Unsure of cause, so the prefixes are duplicated in the code below.

Reply:
I tested this again.  When you import the multiClusterRoutePrefixes it causes this test suite to fail:
`packages/console-shared/src/hooks/__tests__/useUserSettings.spec.ts`

Each test fails with this message:
```
useUserSettings › should return default value for an empty configmap after switching from loading to loaded

    TypeError: user_settings_1.seralizeData is not a function
```

Seems kind of crazy pants to me, but I tested it numerous times and it fails without fail (hahaha) when using the import.

Reply:
Stack trace for failed test:
``` at packages/console-shared/src/hooks/useUserSettings.ts:119:47
      at invokePassiveEffectCreate (node_modules/react-dom/cjs/react-dom.development.js:23487:20)
      at HTMLUnknownElement.callCallback (node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at Object.invokeGuardedCallbackDev (node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at flushPassiveEffectsImpl (node_modules/react-dom/cjs/react-dom.development.js:23574:9)
      at unstable_runWithPriority (node_modules/scheduler/cjs/scheduler.development.js:646:12)
      at runWithPriority$1 (node_modules/react-dom/cjs/react-dom.development.js:11276:10)
      at flushPassiveEffects (node_modules/react-dom/cjs/react-dom.development.js:23447:14)
      at Object.<anonymous>.flushWork (node_modules/react-dom/cjs/react-dom-test-utils.development.js:992:10)
      at flushWorkAndMicroTasks (node_modules/react-dom/cjs/react-dom-test-utils.development.js:1001:5)
      at node_modules/react-dom/cjs/react-dom-test-utils.development.js:1080:11
      ```

Reply:
This is most likely an issue with a jest.mock in the user settings hook unit test.